descriptors/ModuleDescriptor-template.json

```diff
@@ -1438,6 +1438,26 @@
     {
       "id": "contributor-name-types",
       "version": "1.3"
+    },
+    {
+      "id": "source-manager-job-executions",
+      "version": "3.4"
+    },
+    {
+      "id": "organizations.organizations",
+      "version": "1.2"
+    },
+    {
+      "id": "data-import-converter-storage",
+      "version": "1.5"
+    },
+    {
+      "id": "circulation",
```

Comment:
This interface already defined
https://github.com/folio-org/mod-orders/pull/1083/files#diff-447a26c098aa9f236a852f39464c3149d97a871c75f5b9e8ee6e968bdebe319dR1383

Reply:
thanks, removed